### PR TITLE
Functional tests - Add new test 'Check multi invoice'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'Check multi invoice'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27260)

### CODE REVIEW

File: tests/UI/pages/BO/orders/view.js [hunk 1]
```diff
@@ -607,6 +615,32 @@ class Order extends BOBasePage {
...
+    ]);
+    await this.setValue(page, this.editProductPriceInput, price);
+
+    await Promise.all([
+      page.click(this.UpdateProductButton),
+      this.waitForVisibleSelector(page, this.modalConfirmNewPrice),
+    ]);
+
+    await page.click(this.modalConfirmNewPriceSubmitButton);
+    await page.waitForTimeout(2000);
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/view.js]: We should find a solution to delete this.

File: tests/UI/pages/BO/orders/view.js [hunk 2]
```diff
@@ -637,6 +671,71 @@ class Order extends BOBasePage {
...
   }
 
+  /**
+   * Create new invoice
+   * @param page {Page} Browser tab
+   * @returns {Promise<void>}
+   */
+  async createNewInvoice(page) {
+    await this.selectByVisibleText(page, this.addProductInvoiceSelect, 'Create a new invoice');
+  }
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/view.js]: Can we create a more generic function to select from the option with the option by default is `'Create a new invoice'`

**Review by boubkerbribri — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]
Could you please fix conflicts? :)

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**